Resources/input/fsh/ISiKOrganisation.fsh

```diff
@@ -0,0 +1,230 @@
+Profile: ISiKOrganisationFachabteilung
+Parent: Organization
+Id: ISiKOrganisationFachabteilung
+Description: "Dieses Profil beschreibt die Organisationseinheit Fachabteilung innerhalb eines Krankenhauses."
+* insert Meta
+* id MS
+* identifier 1..* MS
+  * ^slicing.discriminator.type = #pattern
+  * ^slicing.discriminator.path = "$this"
+  * ^slicing.rules = #open
+* identifier contains 
+  IKNR 0..1 MS and //TODO: zu prüfen, ob sinnvoll für Fachabteilung als Organisationseinheit eines Krankenhauses
+  BSNR 0..1 MS and //TODO: zu prüfen, ob sinnvoll für Fachabteilung als Organisationseinheit eines Krankenhauses

```

Comment:
Hier am Besten bei der KBV anfragen

Reply:
Profilfestlegung deckt sich mit KBV Basis und VZD, daher eine weitere Anfrage notwendig

Reply:
Das Profil der KBV und des VZDs werden wahrscheinlich einen anderen UseCase haben, ich denke nicht, dass eine BSNR für eine Fachabteilung herausgegeben werden. Zu einer Betriebsstätte gehören 1..n Fachabteilungen. Es sind einfach andere Konzepte die nicht wirklich in Verbindung miteinander stehen.